src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": "^5.5.9|>=7.0.8",
         "symfony/config": "~3.2|~4.0",
+        "symfony/debug": "~2.8|~3.0|~4.0",
```

Comment:
`ExceptionController::showAction()` has a parameter of type `Symfony\Component\Debug\Exception\FlattenException`